### DISCUSSION TITLE
这个报错是怎么回事啊

### DISCUSSION THREAD

**Discussion by ghost** [mikumifa/biliTickerBuy#253] [提问专区]
![QQ截图20240629180437](https://github.com/mikumifa/biliTickerBuy/assets/106471736/78eb03b3-9a7f-4e45-93c4-0ad34336b086)

**Comment by mikumifa** [comment]
1%的验证码错误

**Reply by ghost** [reply]
如何解决

**Reply by mikumifa** [reply]
重新来啊
